src/styles/themes.js

```diff
@@ -13,6 +14,10 @@ const GlobalStyle = createGlobalStyle`
     font-family: ${({ theme }) => theme.font.secondary};
     font-size: 38px;
     font-weight: normal;
+  
+    #root {
```

Comment:
I thought I saw that you fixed the indentation in a commit?!

Reply:
I did fix it! I have no idea why it returned... maybe some weird stuff during rebasing

Reply:
Yeah, that happens :smile: